tools/report.sh

```diff
@@ -81,6 +81,25 @@ if [ -z $namespace ]; then
    usage
 fi
 
+available_namespaces=`$platform get ns -o name`
```

Comment:
It's just a nit, but i think that ` ` is "deprecated" and now we should use the `$()`... but maybe I'm wrong and it's only suggestion...